### PR TITLE
BLOCKS-83 Change Standard Language to 'en'

### PR DESCRIPTION
**Pull request by georgsc**
https://jira.catrob.at/browse/BLOCKS-83
Imported string.xml file from https://github.com/Catrobat/Catroid/blob/develop/catroid/src/main/res/values/strings.xml. Used this xml file to create language reference for 'en'. Set default language in Catblocks to 'en'.

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by georgsc** [comment]
Of course we can leave the 'en-rUS' values away. But then 'en_US' will be unknown and the fallback language 'en' will be used (which is the same). 

So my question is: Should 'en_US' also map to 'i18n/catroid_strings/values-en/strings.xml'. Or should 'en_US' handled as unknown language and the fallback language 'en' is used?

Because the current implementation maps each string.xml file to only one language code. That's why I used the same string.xml file twice. If 'en' and 'en_US' should point to 'i18n/catroid_strings/values-en/strings.xml' I have to do some code modification. But it would be possible of course.

**Comment by wslany** [comment]
> Of course we can leave the 'en-rUS' values away. But then 'en_US' will be unknown and the fallback language 'en' will be used (which is the same).
> 
> So my question is: Should 'en_US' also map to 'i18n/catroid_strings/values-en/strings.xml'. Or should 'en_US' handled as unknown language and the fallback language 'en' is used?
> 
> Because the current implementation maps each string.xml file to only one language code. That's why I used the same string.xml file twice. If 'en' and 'en_US' should point to 'i18n/catroid_strings/values-en/strings.xml' I have to do some code modification. But it would be possible of course.

I'm sorry, I do not understand the consequences. It sounds to me like it would have exactly the same outcome, no? Either the fall-back en is used for en_US (which is 100% correct to do), or both en and en_US point to the same 'i18n/catroid_strings/values-en/strings.xml' (which again is the en version). So basically, it either points implicitly or explicitly to the same en standard version, but in practice there's no difference, right?

**Comment by georgsc** [comment]
Yes, you are right, there would be exactly the same outcome. So I am going to remove the 'en-rUS'  values and using 'en' as fallback language for unknown languages.

**Comment by georgsc** [comment]
Removed duplicate string.xml file for 'en_US'. Current implementation uses 'en' as default language. 
For unknown languages (including 'en_US') the default language will be used. 
So 'en_US' falls back to the default language 'en'. Which is (again) exactly the same.

File: i18n/create_json.js [hunk 1]
```diff
@@ -12,6 +12,7 @@
  *            2020-03-30: [AK] updated to use xml2js module
  *            2020-03-31: [AK] updated code to use default value for missing in current language
  *            2020-04-01: [AK] updated default language to 'values-en-rAU'
+ *            2020-04-10: [GS] included and updated default language to 'values-en-rUS
```
Comment (wslany) [inline i18n/create_json.js:15]:
```suggestion
 *            2020-05-10: [GS] included and updated default language to 'values-en'
```

Comment (georgsc) [reply i18n/create_json.js]: Thanks, well spotted. Changed it to your suggestion.

**Comment by wslany** [comment]
Thanks!

**Pull request merged**